lib/installV1.js

```diff
@@ -185,38 +202,31 @@ function setupDB(done) {
       }
    }
 
-   for (var oo in dbOptions) {
-      if (dbOptions[oo] == "false") {
-         dbOptions[oo] = false;
-      }
-      if (dbOptions[oo] == "true") {
-         dbOptions[oo] = true;
-      }
-   }
-   // console.log(JSON.stringify(dbOptions, null, 4));
+   dbOptions = unstringifyBools(dbOptions);
    configDBCommand
       .run(dbOptions)
       .then(done)
       .catch(done);
 }
 
 function configureNGINX(done) {
+
    var nOptions = {};
+
+   // filter on arguments passed in in the form of "--nginx.arg"
+   for (var i in Options.nginx) {
+      nOptions[i] = Options.nginx[i];
+   }
+   // filter on arguments passed in in the form of "--nginxArg"
    for (var o in Options) {
       if (o.indexOf("nginx") == 0 && o.length > 5) {
          var o2 = o.replace("db", "");
```

Comment:
this looks like it should be: 
```
var o2 = o.replace("nginx", "");
```

Reply:
okay... i thought that might have been the case but I wasn't sure what the intent of the original code was... i can change it and test and see if everything seems to be working

Reply:
I think I'm gonna remove support for "--enableNginx" and only handle "--nginx.enable".  I'm not seeing any evidence of the camel case version of the command in the help ...?